speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/EndExperimentMenu.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+extern void* EndExperimentInit(RaceEndType p_saveWayVersion);
```

Comment:
Fix this linting-error by using #include "Mediator.h", then remove the mediator include from the .cpp file. (The .cpp file will then get the include from the .h file)